cmd/kubectl-testkube/commands/root.go

```diff
@@ -62,6 +63,17 @@ var RootCmd = &cobra.Command{
 	Short: "Testkube entrypoint for kubectl plugin",
 	PersistentPreRun: func(cmd *cobra.Command, args []string) {
 		ui.SetVerbose(verbose)
+
+		if cmd.Name() == "context" {
+			return
+		}
+
+		cfg, err := config.Load()
+		ui.ExitOnError("loading config", err)
+
+		if err = validator.ValidateCloudContext(cfg); err != nil {
+			common.UiCloudContextValidationError(err)
```

Comment:
cli should work without os error code for missed config, like PostRun

Reply:
ok removed it